be/src/exec/exec_node.cpp

```diff
@@ -543,24 +543,7 @@ Status ExecNode::create_vectorized_node(starrocks::RuntimeState* state, starrock
 
 Status ExecNode::create_node(RuntimeState* state, ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs,
                              ExecNode** node) {
-    std::stringstream error_msg;
-
-    VLOG(2) << "tnode:\n" << apache::thrift::ThriftDebugString(tnode);
-
-    if (tnode.use_vectorized) {
-        return create_vectorized_node(state, pool, tnode, descs, node);
-    }
-
-    switch (tnode.node_type) {
-    case TPlanNodeType::SCHEMA_SCAN_NODE:
-        *node = pool->add(new SchemaScanNode(pool, tnode, descs));
-        return Status::OK();
-    default:
-        error_msg << " Don't support old query engine any more";
-        return Status::InternalError(error_msg.str());
-    }
-
-    return Status::OK();
+    return create_vectorized_node(state, pool, tnode, descs, node);
```

Comment:
i think we can remove `create_node`, use `create_vectorized_node ` directly.

Reply:
You are right.  you could submit a PR improve it.